app-crypt/keybase/keybase-1.0.48.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
why not 7?

Reply:
I've just copy-pasted from old existing `*.ebuild` file

Reply:
@nicolasbock I've upgraded it to `7` in my last commits. Any reason for you reverted it?

Reply:
`EAPI` 7 is not supported by the `systemd` eclass.